systemd-firstboot: add page (#10493)
tldr-pages/tldr: pages/linux/systemd-firstboot.md

# systemd-firstboot

> Initialize basic system settings on or before the first boot-up of a system.
> More information: <https://www.freedesktop.org/software/systemd/man/systemd-firstboot.html>.

> - Set the root directory for the basic system settings initialization:

Reviewer: ```suggestion
- Operate on the specified directory instead of the root directory of the host system:
```

> `sudo systemd-firstboot --root={{root_directory_path}}`

Reviewer 2: ```suggestion
`sudo systemd-firstboot --root={{path/to/root_directory}}`
```

- Set the system keyboard layout:

`sudo systemd-firstboot --keymap={{keymap}}`

- Set the system hostname:

`sudo systemd-firstboot --hostname={{hostname}}`

- Set the root user's password:

`sudo systemd-firstboot --root-password={{password}}`

- Prompt the user interactively for a specific basic setting:

`sudo systemd-firstboot --prompt={{setting}}`

- Force writing configuration even if the relevant files already exist:

`sudo systemd-firstboot --force`

> - Remove all existing files that are configured by systemd-firstboot:

Reviewer 2: ```suggestion
- Remove all existing files that are configured by `systemd-firstboot`:
```
